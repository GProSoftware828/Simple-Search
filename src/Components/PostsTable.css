.title-space {
	width: 20%;
}

.body-space {
	width: 80%;
}

.table {
	width: 100%;
	padding: 10px;
	border: 1px solid black;
}

#rows:nth-child(even) {
	background-color: #eeeeee;
}