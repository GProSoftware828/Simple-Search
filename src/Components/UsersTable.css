.name-space {
	width: 30%;
}

.email-space {
	width: 30%;
}

.city-space {
	width: 20%;
}

.company-space {
	width: 20%;
}

.table {
	width: 100%;
	padding: 10px;
	border: 1px solid black;
}

#rows:nth-child(even) {
	background-color: #eeeeee;
}