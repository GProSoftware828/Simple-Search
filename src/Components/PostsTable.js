import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './PostsTable.css';

export class PostsTable extends Component {
  render() {
    return (
      <div
        data-test="PostComponent"
				id="rows"
      >
        <table className="table">
					<tbody>
						<tr>
							<td className="title-space">
								Title
							</td>
							<td className="body-space">
							Body
							</td>
						</tr>
						<tr>
							<td class="title-space">
								{this.props.title}
							</td>
							<td className="body-space">
								{this.props.body}
							</td>
						</tr>
					</tbody>
        </table>
      </div>
    );
  }
}

PostsTable.propTypes = {
  title: PropTypes.string,
	body: PropTypes.string
};

export default PostsTable;
