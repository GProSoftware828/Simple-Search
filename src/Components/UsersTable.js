import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './UsersTable.css';

export class UsersTable extends Component {
  render() {
    return (
      <div
        onClick={this.props.click}
        data-testid="users-table"
				id="rows"
      >
        <table className="table">
					<tbody>
						<tr>
							<td className="name-space">
								Name
							</td>
							<td className="email-space">
							Email
							</td>
							<td className="city-space">
							City
							</td>
							<td className="company-space">
			         Company
							</td>
						</tr>
						<tr>
							<td class="name-space">
								{this.props.name}
							</td>
							<td className="email-space">
								{this.props.email}
							</td>
							<td className="city-space">
								{this.props.address}
							</td>
							<td className="company-space">
								{this.props.company}
							</td>
						</tr>
					</tbody>
        </table>
      </div>
    );
  }
}

UsersTable.propTypes = {
  name: PropTypes.string,
	email: PropTypes.string,
	address: PropTypes.string,
	company: PropTypes.string
};

export default UsersTable;
