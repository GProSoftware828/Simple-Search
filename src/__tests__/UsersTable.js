import { UsersTable } from '../Components/UsersTable';
import { render, fireEvent, cleanup } from '@testing-library/react';

test('Should be the same UsersTable- snapshot', () => {
	expect(UsersTable).toMatchSnapshot();
})