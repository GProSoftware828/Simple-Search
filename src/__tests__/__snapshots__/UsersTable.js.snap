// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be the same UsersTable- snapshot 1`] = `[Function]`;
